### DISCUSSION TITLE
How to create a UV texture image from Vertex color textures

### DISCUSSION THREAD

**Discussion by elcronos** [facebookresearch/pytorch3d#545] [Q&A]
I was able to backpropagate the vertex color texture using a custom loss function. Now, I'd like to save the texture so I can exported an used it in Blender. How can I save the vertex color as the texture? If it's not possible, Can I create a UV texture based on the Vertex colors?

Regards,

Camilo
